src/NServiceBus.Transport.SQS/Configure/SqsTransport.cs

```diff
@@ -57,6 +57,11 @@ internal set
         /// </summary>
         public string QueueNamePrefix { get; set; }
 
+        /// <summary>
+        /// Disable native delayed delivery infrastructure
```

Comment:
Maybe be more explicit here, for example:

```suggestion
        /// Disable native delayed delivery infrastructure. This property is a workaround allowing ServiceControl to disable delayed deliveries when setting up message pumps for error and audit queues
```

Reply:
No, I think there are many good reasons to just not have it enabled. Especially when not using delayed delivery at all or with native integration